statsmodels/compat/__init__.py

```diff
@@ -1 +1,23 @@
-from .python import *
```

Comment:
Can't you just leave this in and noqa it?  Or do you get other flake errors?

Reply:
That would be allowed.  In this case changing it surfaced the fact that `urlretrieve` was defined in the py3 section but not the py2 section.  Not that we actually need `urlretrieve`, but the explicitness can be useful.